plugins/slack_app/test/models/slack_message_test.rb

```diff
@@ -0,0 +1,81 @@
+require_relative '../test_helper'
+
+SingleCov.covered!
+
+describe SlackMessage do
+  let(:deploy) { deploys(:succeeded_test) }
+  let(:msg) { SlackMessage.new deploy }
+  let(:body) { msg.message_body }
+  let(:user1) { users(:deployer) }
+  let(:user2) { users(:deployer_buddy) }
+  let(:user1_identifier) { slack_identifiers(:user1) }
+  let(:user2_identifier) { slack_identifiers(:user2) }
+  before do
+    user1_identifier.update_column(:user_id, deploy.user.id)
+    user2_identifier.update_column(:user_id, user2.id)
+  end
+
+  describe 'a deploy waiting for a buddy' do
```

Comment:
this should all sit in `describe "#message_body"` ... and then also have a test for deliver ?